sycl/plugins/hip/pi_hip.cpp

```diff
@@ -5345,6 +5384,9 @@ pi_result piPluginInit(pi_plugin *PluginInit) {
   _PI_CL(piextUSMEnqueueFill2D, hip_piextUSMEnqueueFill2D)
   _PI_CL(piextUSMEnqueueMemset2D, hip_piextUSMEnqueueMemset2D)
   _PI_CL(piextUSMGetMemAllocInfo, hip_piextUSMGetMemAllocInfo)
+  // Device variable
```

Comment:
```suggestion
  // Device global variable
```

? I'm leaving such comments, because we can't access an arbitrary variable allocated on a device, I just want the API and description to better match the extension